wagtail/admin/templates/wagtailadmin/skeleton.html

```diff
@@ -4,35 +4,36 @@
 {% get_current_language_bidi as LANGUAGE_BIDI %}
 <html lang="{{ LANGUAGE_CODE }}" dir="{% if LANGUAGE_BIDI %}rtl{% else %}ltr{% endif %}" class="{% admin_theme_classname %}">
     <head>
-        <meta charset="utf-8" />
-        <title>{% block titletag %}{% endblock %} - {% block branding_title %}Wagtail{% endblock %}</title>
-        <meta name="description" content="" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta name="robots" content="noindex" />
-
-        {% block css %}{% endblock %}
-
-        {% block branding_favicon %}{% endblock %}
+        <meta charset="utf-8"/>
+        <title>
+            {% block titletag %}{% endblock %}
+            -
+            {% block branding_title %}Wagtail
+            {% endblock %}
+        </title>
+        <meta name="description" content=""/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        <meta name="robots" content="noindex"/> {% block css %}{% endblock %}{% block branding_favicon %}{% endblock %}
     </head>
     {% sidebar_collapsed as sidebar_collapsed %}
-    <body id="wagtail" class="{% block bodyclass %}{% endblock %} {% if sidebar_collapsed %}sidebar-collapsed{% endif %} {% if messages %}has-messages{% endif %}">
-        <div data-sprite></div>
+    <body id="wagtail" class="{% block bodyclass %}{% endblock %} {% if sidebar_collapsed %}sidebar-collapsed{% endif %} {% if messages %}has-messages{% endif %}" data-controller="w-init" data-w-init-ready-class="ready">
+        <div data-sprite="data-sprite"></div>
```

Comment:
Data-sprite as it is written currently is correct. Best to leave that as is I think.